hyperspy/tests/io/test_hdf5.py

```diff
@@ -289,6 +289,25 @@ def tearDown(self):
         remove(self.filename)
 
 
+class TestAxesConfiguration:
+
+    def setUp(self):
+        self.filename = 'testfile.hdf5'
+        s = BaseSignal(np.zeros((2, 2, 2, 2, 2)))
+        s.axes_manager.signal_axes[0].navigate = True
+        s.axes_manager.signal_axes[0].navigate = True
```

Comment:
You are right, this wasn't documented. I have just made some improvements to this part of the docs.